src/CommonDropdown.php

```diff
@@ -202,22 +202,32 @@ public function defineTabs($options = [])
         return $ong;
     }
 
-
     public function displayHeader()
     {
-
-        if (empty($this->third_level_menu)) {
-            $this->third_level_menu = $this->getType();
-        }
-        Html::header(
-            $this->getTypeName(Session::getPluralNumber()),
-            '',
-            $this->first_level_menu,
-            $this->second_level_menu,
-            $this->third_level_menu
+        Toolbox::deprecated(
+            "This method is deprecated. Use displayCentralHeader() instead"
         );
+        static::displayCentralHeader();
     }
 
+    public static function displayCentralHeader(
+        ?string $title = null,
+        ?array $menus = null
+    ): void {
+        if (is_null($menus)) {
+            $dropdown = new static();
+
+            $menus = [
+                $dropdown->first_level_menu,
+                $dropdown->second_level_menu,
+                empty($dropdown->third_level_menu) ?
+                    $dropdown->getType() :
+                    $dropdown->third_level_menu
```

Comment:
The Elvis operator could handle this.
```suggestion
                $dropdown->third_level_menu ?: $dropdown->getType()
```